target/start-mailserver.sh

```diff
@@ -543,13 +543,25 @@ function _setup_dovecot
 
   # set mail_location according to mailbox format
   case "${DOVECOT_MAILBOX_FORMAT}" in
-    sdbox|mdbox|maildir )
+    sdbox|mdbox )
       _notify 'inf' "Dovecot ${DOVECOT_MAILBOX_FORMAT} format configured"
       sed -i -e 's/^mail_location = .*$/mail_location = '"${DOVECOT_MAILBOX_FORMAT}"':\/var\/mail\/%d\/%n/g' /etc/dovecot/conf.d/10-mail.conf
+
+      _notify 'inf' "Creating cron job for dbox purge"
+      cat <<EOF >/etc/cron.d/dovecot-purge
+#
```

Comment:
This works, but personally I would prefer to keep the file separate and just make a copy here. That keeps the start script a bit simpler and makes it easier to change the cronjob.

Reply:
Where is the best place to put a crontab file? It could be in `target/dovecot/`.

Reply:
Sounds good to me.